src/Service/Siteminder.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\siteminder\Service\Siteminder.
+ */
+
+namespace Drupal\siteminder\Service;
```

Comment:
quite new to me. We add managers, plugins, factory, repository and make them as service. Don't think we do service as namespace, unless it's functionally need to be.